src/models/calendar-restful-model-collection.ts

```diff
@@ -105,4 +105,68 @@ export default class CalendarRestfulModelCollection extends RestfulModelCollecti
         return Promise.reject(err);
       });
   }
+
+  consecutiveAvailability(
+    options: {
+      emails: Array<{
+        email: string;
+        name: string;
+      }>;
+      duration: number;
+      interval: number;
+      start_time?: string;
+      startTime?: string;
+      end_time?: string;
+      endTime?: string;
+      free_busy?: Array<{
+        emails: string;
+        object: string;
+        time_slots: Array<{
+          object: string;
+          status: string;
+          start_time: string;
+          end_time: string;
+        }>;
+      }>;
+      open_hours: Array<{
+        emails: Array<{
+          email: string;
+          name: string;
+        }>;
+        days: string[];
+        timezone: string;
+        start: string;
+        end: string;
+        object_type: string;
+      }>;
+    },
+    callback?: (error: Error | null, data?: { [key: string]: any }) => void
+  ) {
+    return this.connection
+      .request({
+        method: 'POST',
+        path: `/calendars/availability/consecutive`,
+        body: {
+          emails: options.emails,
+          duration_minutes: options.duration,
+          interval_minutes: options.interval,
+          start_time: options.startTime || options.start_time,
+          end_time: options.endTime || options.end_time,
+          free_busy: options.free_busy || [],
+          open_hours: options.open_hours,
+        },
```

Comment:
You can also add `buffer` as a potential input to the call although it's not in the docs yet. It takes a non-negative integer or float representing the buffer minutes to add to busy time. 

If, for example, you send `buffer: 30`, there will be a 30 minute buffer before and after each individual's existing meetings/busy time that will be counted as busy.

Reply:
Added in, thank you!